VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -186,8 +184,17 @@ public static CharSequence trim(CharSequence s) {
         return s.subSequence(start, end);
     }
 
-    public static CharSequence removeDuplicateWhitespace(CharSequence text) {
-        return PATTERN_DUPLICATE_WHITESPACE.matcher(text).replaceAll(" ");
+    public static CharSequence removeDuplicateWhitespaces(CharSequence text) {
```

Comment:
'whitespace' is an uncountable noun, so there's no need to pluralize it: https://english.stackexchange.com/questions/25368/what-is-the-plural-form-of-whitespace